File: Java/peakIndexInArray.java
class Solution {
    public int peakIndexInMountainArray(int[] A) {
        int max = 0;
        int index = 0;

        for (int i = 0; i < A.length; i++) {
            if (A[i] > max ) {
                max = A[i];
                index = i;
            }
        }
        return index;
    }
}

File: Easy/Golang/peakIndexInArray.go
import "jmt"

func peakIndexInMountainArray(A []int) int {

    var max = 0
    var index = 0

    for i := 0; i < len(A); i++ {
        if (A[i] > max) {
            max = A[i]
            index = i
        }
    }
    return index
}
